src/.vuepress/theme/components/AlgoliaSearchBox.vue

```diff
@@ -36,135 +30,124 @@ export default {
 
   mounted () {
     this.initialize(this.options, this.$lang)
-    this.placeholder = this.$site.themeConfig.searchPlaceholder || ''
   },
 
   methods: {
+    getRelativePath (absoluteUrl) {
+      const { pathname, hash } = new URL(absoluteUrl)
+      const url = pathname.replace(this.$site.base, '/') + hash
+
+      return url
+    },
     initialize (userOptions, lang) {
       Promise.all([
-        import(/* webpackChunkName: "docsearch" */ 'docsearch.js/dist/cdn/docsearch.min.js'),
-        import(/* webpackChunkName: "docsearch" */ 'docsearch.js/dist/cdn/docsearch.min.css')
+        import(
+          /* webpackChunkName: "docsearch" */ '../node_modules/@docsearch/js'
```

Comment:
The build system doesn't seem to accept external imports starting with `@` so I referenced to `node_modules` here, but it doesn't build. Feel free to update this 🙂

Reply:
Fixed in a6d5adf810a8bce75e23c8d17eff5e88a6bd79cb.